Godeps/_workspace/src/k8s.io/kubernetes/cmd/kubelet/app/server.go

```diff
@@ -148,6 +148,9 @@ type KubeletServer struct {
 
 	KubeApiQps   float32
 	KubeApiBurst int
+
+	// Pull images one at a time.
+	SerializeImagePulls bool
```

Comment:
fixed in last UPSTREAM commit, the upstream had been using an incorrect default.
https://github.com/kubernetes/kubernetes/pull/16749